portfolio/src/main/webapp/script.js

```diff
@@ -75,3 +75,70 @@ function addQuoteToDom(quote) {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+const google = window.google;
+window.onload = createMap;
```

Comment:
I think you can only assign to `window.onload` once. This will overwrite the `onload = getMessage` part.

What you want is to do something like this:
```js
// All the things to do when starting up.
window.onload = function() {
  getMessage();
  createMap();
};
```

Reply:
Done!

Reply:
did you push?